build.gradle

```diff
@@ -398,7 +398,6 @@ task createApiDocsRedirect {
 }
 
 task publishDocs {
-  dependsOn 'closeAndReleaseRepository'
```

Comment:
This is the task defined by the[ 'io.codearte.nexus-staging' plugin](https://github.com/Codearte/gradle-nexus-staging-plugin) which is not listed in the build file but it should work in the runtime.